src/coreComponents/constitutive/solid/SolidBase.hpp

```diff
@@ -643,6 +643,15 @@ class SolidBase : public constitutive::ConstitutiveBase
     return m_density;
   }
 
+  /**
+   * @brief Full accessor for density.
+   * @return Accessor
+   */
+  array2d< real64 > & getDensityFull()
```

Comment:
Why do you need this? I would avoid giving the possibility to access the array directly. I a resize is necessary it should be the case for the entire `objectManager` (probably a subRegion in this case) and all wrappers should be resized at once.

Reply:
I can't remember why I added this, so I'll remove it.